assets/l10n/intl_en.arb

```diff
@@ -2956,5 +2956,14 @@
   "failToSend": "Failed to send, please try again",
   "showLess": "Show Less",
   "showMore": "Show More",
-  "sharedMediaAndFiles": "Shared media and files"
+  "sharedMediaAndFiles": "Shared media and files",
+  "groupInformation": "Group information",
+  "linkInvite": "Link invite",
+  "noDescription": "No description",
+  "description": "Description",
+  "groupName": "Group name",
+  "descriptionHelper": "You can provide an optional description for your group.",
+  "errorUploadingGroupName": "Error uploading group name.",
+  "errorUploadingGroupDescription": "Error uploading group description.",
+  "errorUploadingGroupAvatar": "Error uploading group avatar."
```

Comment:
Please ask designer for the common error. But we need to show the permission error in the case of no permission. (dont use the same error message for all)